app/controllers/concerns/inventory_service_helper.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# Module to support controller integrations with the host inventory service
+module InventoryServiceHelper
```

Comment:
This whole module was moved from base_mutation.rb, nearly unchanged except for renaming the `find_hosts` method to `add_inventory_hosts` to be explicit that it's related to the inventory.